@xen-orchestra/lite/src/components/RangeInput.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
+
+interface Props extends Omit<React.ComponentPropsWithoutRef<'input'>, 'type'> {}
+
+const RangeInput: React.FC<Props> = React.memo(props => <input {...props} type='range' />)
```

Comment:
`React.FC` is discouraged, use the normal function typing instead.
https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components